pkg/pipeline/controller.go

```diff
@@ -297,7 +303,12 @@ func (c *Controller) UpdateStream(ctx context.Context, req *livekit.UpdateStream
 
 	// remove stream outputs
 	for _, rawUrl := range req.RemoveOutputUrls {
-		url, _, err := c.ValidateUrl(rawUrl, types.OutputTypeRTMP)
+		outputType := types.OutputTypeRTMP
+		if strings.HasPrefix(rawUrl, "srt://") {
```

Comment:
A `switch` statement may work better here too.

Reply:
schanged to switch case